addons/packages/vsphere-cpi/1.23.0-alpha.1/bundle/config/values.yaml

```diff
@@ -3,6 +3,7 @@
 
 ---
 vsphereCPI:
+  mode: vsphereCPI
```

Comment:
Can we update the explanation of this field in README?

Reply:
sure, updated those fields in readme